addOns/websocket/src/main/zapHomeFiles/scripts/templates/websocketpassive/Application Error Scanner Regex.js

```diff
@@ -0,0 +1,69 @@
+// * This Script analyzes incoming websocket messages for error messages with a set of regular expressions
+
+// * Based on org.zaproxy.zap.extension.pscanrules.ApplicationErrorScanner
+// * Application error strings are equal to (characters '/' is escaped -> '//'):
```

Comment:
The patterns actually use the opposite slash `\` > `\\`

Reply:
Comment resolved